diff --git a/node_modules/@markdown-confluence/lib/dist/MarkdownTransformer/index.js b/node_modules/@markdown-confluence/lib/dist/MarkdownTransformer/index.js
index e822d3f..0b46faa 100644
--- a/node_modules/@markdown-confluence/lib/dist/MarkdownTransformer/index.js
+++ b/node_modules/@markdown-confluence/lib/dist/MarkdownTransformer/index.js
@@ -121,8 +121,9 @@ const mdToPmMapping = {
     },
 };
 const md = MarkdownIt("commonmark", {
-    html: false,
-    linkify: true,
+	html: false,
+	linkify: true,
+	breaks: true,
 });
 md.enable([
     // Process html entity - &#123;, &#xAF;, &quot;, ...
diff --git a/node_modules/@markdown-confluence/lib/src/ConniePageConfig.ts b/node_modules/@markdown-confluence/lib/src/ConniePageConfig.ts
index dbbe9b8..7fa37bf 100644
--- a/node_modules/@markdown-confluence/lib/src/ConniePageConfig.ts
+++ b/node_modules/@markdown-confluence/lib/src/ConniePageConfig.ts
@@ -196,7 +196,7 @@ export const conniePerPageConfig: ConfluencePerPageConfig = {
 		},
 	},
 	pageId: {
-		key: "connie-page-id",
+		key: "confluence-page-id",
 		default: undefined,
 		inputType: "text",
 		inputValidator: (value) => {
